src/commands/pull.yml

```diff
@@ -1,6 +1,10 @@
 description: Pull one or more Docker images from a registry
 
 parameters:
+  step-name:
```

Comment:
I think we can do without the parameter for step name.

Reply:
Done.